docs/src/api/class-route.md

```diff
@@ -226,6 +226,13 @@ is resolved relative to the current working directory.
 
 [APIResponse] to fulfill route's request with. Individual fields of the response (such as headers) can be overridden using fulfill options.
 
+### option: Route.fulfill.cors
+- `cors` <[boolean]>
```

Comment:
WDYT about `cors: 'allow' | 'deny' | 'none'`?

Also, here is an example of configuring cors on the server side (similar to our use case): https://expressjs.com/en/resources/middleware/cors.html